pyproject.toml

```diff
@@ -66,6 +66,7 @@ xmltodict = { version = ">=0.12.0", optional = true }
 
 [tool.poetry.extras]
 markup = [ "anymarkup-core", "xmltodict" ]
+testing = ["pytest-container"]
```

Comment:
Let's change this to `integration_tests`

Reply:
done